Source/WebCore/rendering/updating/RenderTreeBuilderInline.cpp

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2018 Apple Inc. All rights reserved.
+ * Copyright (C) 2018-2024 Apple Inc. All rights reserved.
+ * Copyright (C) 2013 Google Inc. All rights reserved.
```

Comment:
I understand the desire to give credit to all contributors, but a single clause in an if statement is not really enough for a copyright update.

Reply:
Sure, will revert in next push and submit for 'merge-queue'.

Reply:
Done and pushed newer commit with following removed (and rebased). :-)